teamscale-gradle-plugin/build.gradle

```diff
@@ -25,9 +25,8 @@ dependencies {
 	implementation project(':teamscale-client')
 	implementation project(':report-generator')
 	implementation gradleApi()
-	implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.3.41'
-	implementation 'org.eclipse.jgit:org.eclipse.jgit:5.0.2.201807311906-r'
-	implementation 'com.squareup.retrofit2:converter-moshi:2.5.0'
+	implementation 'org.eclipse.jgit:org.eclipse.jgit:5.11.1.202105131744-r'
```

Comment:
Seems to be fine. Solves CVE-2020-13956 (HttpClient).

Reply:
Thanks for re-checking!